baselines/deprecatedtest/src/v1/deprecated_service_client.ts.baseline

```diff
@@ -150,6 +150,8 @@ export class DeprecatedServiceClient {
     // Load the applicable protos.
     this._protos = this._gaxGrpc.loadProtoJSON(jsonProtos);
 
+    const protoFilesRoot = this._gaxModule.protobuf.Root.fromJSON(jsonProtos);
```

Comment:
Looks like it's unused in this case and will cause warning. Maybe it should be under "if LRO" in the template?